priv/repo/migrations/20240130130617_create_software_update_settings.exs

```diff
@@ -0,0 +1,26 @@
+defmodule Trento.Repo.Migrations.CreateSoftwareUpdateSettings do
+  use Ecto.Migration
+
+  def change do
+    settings_name = "software_update_settings"
+
+    create table(:software_update_settings, primary_key: false) do
+      add :name, :string, primary_key: true, default: settings_name
```

Comment:
Maybe this could be a UUID as we do for the settings table? Just a nitpick, I don't know how I feel about having a primary key that is a plain string :grimacing: 

Also, I don't think we need the `default` here since we have the constraint to keep this to one record only and the initial insert already populates the record with the value we care about

Reply:
ok, I will make it a uuid if that makes feel you better :smile: 

However, let's keep the default, it just comes handy when operating with that record. Is that ok?